pkg/distro/fedora/images.go

```diff
@@ -353,9 +353,9 @@ func imageInstallerImage(workload workload.Workload,
 		img.UnattendedKickstart = instCust.Unattended
 	}
 
-	// Enable anaconda-webui for Fedora > 38
+	// Enable anaconda-webui for Fedora > 39
 	distro := t.Arch().Distro()
-	if !common.VersionLessThan(distro.Releasever(), "38") {
+	if !common.VersionLessThan(distro.Releasever(), "39") {
```

Comment:
so the comment claims this means `> 39`, but the name doesn't seem to match. "not less than X" should be equivalent to "X or greater", no? 39 is not less than 39. I would think these should all be 40, not 39, at least on a naive reading of the name of the function.

Reply:
Thank you, yes. The function name is not ideal and you are correct.